### PR TITLE
fix: add documentation links in help

### PR DESCRIPTION
**Pull request by Sanket322**
## Clicking Help Now will display the documentation link specific to the doctype.

Like here Clicking GSTR-1 Beta will redirect you to its documentation.

Documentation

<sub><a href="https://huly.app/guest/resilienttech?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJsaW5rSWQiOiI2NzRmZTY1Y2QyZWY0Y2JjYzQ2ZTFlNTQiLCJndWVzdCI6InRydWUiLCJlbWFpbCI6IiNndWVzdEBoYy5lbmdpbmVlcmluZyIsIndvcmtzcGFjZSI6Inctc21pdHZvcmEyMDMtcmVzaWxpZW50dGVjLTY2N2U0MjkxLWEwNWMwNjY4N2EtNjM4MjY3In0.XQ_GSRxU4m2V3plr9mCXXSQndwEVnlt5HSSC7qDpLrA">Huly&reg;: <b>IC-2936</b></a></sub>

### CODE REVIEW

**Comment by vorasmit** [comment]
Which versions are supported?

**Comment by Sanket322** [comment]
> Which versions are supported?
Version 14 is supported.

File: india_compliance/public/js/help_links.js [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+
+//GST Settings
+frappe.help.help_links["Form/GST Settings"] = [
+    {
+        label: "Setting Up GST accounts",
+        url: docsUrl + "configuration/gst_setup#gst-accounts"
+    },
+    {
+        label: "GSTIN Verification",
+        url: docsUrl + "miscellaneous/gstin_verification",
```
Comment (vorasmit) [inline india_compliance/public/js/help_links.js]: Setting up e-Invoice creds

File: india_compliance/public/js/help_links.js [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    },
+];
+
+//Company
+frappe.help.help_links["Form/Company"] = [
+    {
+        label: "Print Format",
```
Comment (vorasmit) [inline india_compliance/public/js/help_links.js]: Print Settings => add more context from doctype

File: india_compliance/public/js/help_links.js [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+//Doctypes
+//Sales Invoice
+if (!frappe.help.help_links["Form/Sales Invoice"]) {
+    frappe.help.help_links["Form/Sales Invoice"] = [];
+}
+
+frappe.help.help_links["Form/Sales Invoice"].push(
+    {
+        label: "Sales Transaction",
```
Comment (vorasmit) [inline india_compliance/public/js/help_links.js]: Not required

File: india_compliance/public/js/help_links.js [hunk 4]
```diff
@@ -0,0 +1,194 @@
...
+        url: docsUrl + "ewaybill-and-einvoice/generating_e_waybill",
+    },
+    {
+        label: "e-Invoice",
+        url: docsUrl + "ewaybill-and-einvoice/generating_e_invoice",
+    },
+);
+
+frappe.help.help_links["List/Sales Invoice"].push({
+    label: "Sales Transaction",
```
Comment (vorasmit) [inline india_compliance/public/js/help_links.js]: Not required

File: india_compliance/public/js/help_links.js [hunk 5]
```diff
@@ -0,0 +1,194 @@
...
+    url: docsUrl + "ewaybill-and-einvoice/generating_e_waybill",
+})
+
+//Purchase Receipt
+frappe.help.help_links["Form/Purchase Receipt"] = [
+    {
+        label: "e-Waybill",
+        url: docsUrl + "ewaybill-and-einvoice/generating_e_waybill",
+    }
+]
```
Comment (vorasmit) [inline india_compliance/public/js/help_links.js]: Ignored

File: india_compliance/public/js/help_links.js [hunk 6]
```diff
@@ -0,0 +1,194 @@
...
+
+//Stock Entry
+frappe.help.help_links["Form/Stock Entry"].push({
+    label: "e-Waybill",
+    url: docsUrl + "ewaybill-and-einvoice/generating_e_waybill",
```
Comment (vorasmit) [inline india_compliance/public/js/help_links.js]: Blog

File: india_compliance/public/js/help_links.js [hunk 7]
```diff
@@ -0,0 +1,194 @@
...
+    {
+        label: "GST Purchase Register",
+        url: docsUrl + "gst-reports/miscellaneous_reports#gst-purchase-register-beta-report",
+    },
+];
+
+//Purchase Reconciliation
+frappe.help.help_links["Form/Purchase Reconciliation Tool"] = [
+    {
+        label: "Setup Purchase Reconciliation Tool",
```
Comment (vorasmit) [inline india_compliance/public/js/help_links.js]: Ignore this

File: india_compliance/public/js/help_links.js [hunk 8]
```diff
@@ -0,0 +1,194 @@
...
+        url: docsUrl + "purchase-reconciliation/purchase_reconciliation_setup",
+    },
+    {
+        label: "Reconciling Purchase",
+        url: docsUrl + "purchase-reconciliation/reconciling_purchase",
+    },
+    {
+        label: "Auto Reconcile",
```
Comment (vorasmit) [inline india_compliance/public/js/help_links.js]: Ignore

File: india_compliance/public/js/help_links.js [hunk 9]
```diff
@@ -0,0 +1,194 @@
...
+        url: docsUrl + "purchase-reconciliation/auto_reconcile",
+    },
+];
+
+//Miscellaneous
+frappe.help.help_links["Form/Audit Trail"] = [
+    {
+        label: "Audit Trail",
```
Comment (vorasmit) [inline india_compliance/public/js/help_links.js:141]: Add this to audit trail report

**Pull request merged**